src/chi/components/header/header.scss

```diff
@@ -568,17 +595,26 @@
           padding-left: 2rem;
           padding-right: 2rem;
         }
+      }
+    }
+  }
 
-        &::before {
-          border-color: $header-navbar-dropdown-arrow-color transparent transparent transparent;
-          border-style: solid;
-          border-width: 0.625rem 0.75rem 0 0.75rem;
-          content: '';
-          height: 1rem;
-          left: 1rem;
-          position: absolute;
-          top: 0;
-          width: 1rem;
+  &.-lg,
+  &.-xl {
+    .chi-header__navbar {
+      .chi-dropdown__menu {
+        .chi-dropdown__menu-item {
+          &::before {
```

Comment:
&:before must be on dropdown__menu selector and not target to menu items. It is causing it renders several times